cosi-examples/bucketclaim-deletion-policy.yaml

```diff
@@ -0,0 +1,8 @@
+kind: BucketClaim
+apiVersion: objectstorage.k8s.io/v1alpha1
+metadata:
+  name: bucket-claim-auto-delete-s3-bucket
+spec:
+  bucketClassName: delete-bucket-class
+  protocols:
+  - s3
```

Comment:
```suggestion
    - s3
```

indent

Reply:
its up for debate, I have usually seen (most common) without indent when using `-`.
I would prefer to follow the industry norm here, unless we have a specific standard or reason.

Technically both are correct though.